src/placeholders.js

```diff
@@ -19,25 +19,53 @@ import { toCamelCase } from './utils.js';
  */
 // eslint-disable-next-line import/prefer-default-export
 export async function fetchPlaceholders(prefix = 'default') {
+  const overrides = getMetadata('placeholders') || getMetadata('root')?.replace(/\/$/, '/placeholders.json') || '';
+  const [fallback, override] = overrides.split('\n');
   window.placeholders = window.placeholders || {};
+
   if (!window.placeholders[prefix]) {
     window.placeholders[prefix] = new Promise((resolve) => {
-      fetch(`${prefix === 'default' ? '' : prefix}/placeholders.json`)
-        .then((resp) => {
+      const url = fallback || `${prefix === 'default' ? '' : prefix}/placeholders.json`;
+      Promise.all([fetch(url), override ? fetch(override) : Promise.resolve()])
+        // get json from sources
+        .then(async ([resp, oResp]) => {
           if (resp.ok) {
-            return resp.json();
+            if (oResp?.ok) {
+              return Promise.all([resp.json(), await oResp.json()]);
+            }
+            return Promise.all([resp.json(), {}]);
           }
-          return {};
-        }).then((json) => {
+          return [{}];
+        })
+        // process json from sources
+        .then(([json, oJson]) => {
           const placeholders = {};
-          json.data
-            .filter((placeholder) => placeholder.Key)
-            .forEach((placeholder) => {
-              placeholders[toCamelCase(placeholder.Key)] = placeholder.Text;
-            });
+
+          const allKeys = new Set([
+            ...(json.data?.map(({ Key }) => Key) || []),
+            ...(oJson?.data?.map(({ Key }) => Key) || []),
+          ]);
+
+          allKeys.forEach((Key) => {
+            if (!Key) return;
+            const keys = Key.split('.');
+            const originalValue = json.data?.find((item) => item.Key === Key)?.Value;
+            const overrideValue = oJson?.data?.find((item) => item.Key === Key)?.Value;
```

Comment:
This is an important change here: you decided to change the placeholders sheet from Key | Text to Key | Value (see also doc https://www.aem.live/docs/placeholders).
While it might make sense, this is a critical change which is not documented in your PR.

Reply:
@kptdobe You are right. It might be a misused in the Commerce Boilerplate that needs to be corrected on that end. I'll revert that change.

Reply:
Done. Reverted the change.